src/build_tools.md

```diff
@@ -0,0 +1,122 @@
+# Build Tools
+
+This section covers "build-time" tooling, or code that is run prior to compiling a crate's source code.
+Conventionally, build-time code lives in a `build.rs` file and is commonly referred to as a "build script".
+Common use cases include `rust` code generation and compilation of bundled `C/C++/asm` code.
+See `crates.io`'s [documentation on the matter][build-script-docs] for more information.
+
+Notes:
```

Comment:
I would remove the notes section here. The play button will not work anyway as these examples will be `no_run` anyway.